openlibrary/templates/merge_request_table/table_row.html

```diff
@@ -0,0 +1,79 @@
+$def with (request, username, can_merge)
+	
+	$ request_title = request.get('title', _('An untitled work') if request['mr_type'] != 2 else _('An unnamed author'))
+	$ comments = request.get('comments', {}).get('comments', [])
+	$ is_open = request['status'] == 1
+	$ review_url = "%s&mrid=%s" % (request['url'], request['id'])
+	$ is_submitter = username == request['submitter']
```

Comment:
```suggestion
	$ request_title = request.get('title', _('An untitled work') if request['mr_type'] != 2 else _('An unnamed author'))
$ comments = request.get('comments', {}).get('comments', [])
$ is_open = request['status'] == 1
$ review_url = "%s&mrid=%s" % (request['url'], request['id'])
$ is_submitter = username == request['submitter']
```
There's no reason for this to be indented.  Make the change outlined above, and also remove one level of indentation from lines 9 to the end of the file.